### DISCUSSION TITLE
Webpack with less-loader seems to not be finding any Less files

### DISCUSSION THREAD

**Discussion by justinfarrelldev** [webpack/webpack#16073] [Q&A] [answered]
I have checked quite a few issues and discussions and have not been able to resolve my problem just yet. Probably the closest to my issue that I found was this: https://github.com/webpack-contrib/less-loader/issues/45, but when I installed Less it didn't change anything. 

Whenever I run my Webpack dev server (with `less-loader`, `css-loader` and `style-loader` in that order), it simply doesn't find any Less files. If I change the contents of a Less file, it doesn't reload (which is why I'm thinking it's simply not finding them).

Here is the relevant bit of configuration: 
```
      {
        test: /\.less$/i,
        include: [path.join(__dirname, 'styles'), path.join(__dirname, 'src')],
        use: [
          process.env.DEVELOPMENT === 'true'
            ? 'style-loader'
            : MiniCssExtractPlugin.loader,
          'css-loader',
          'less-loader',
        ],
      },
```  

(My `.env` file is set to development mode)

I also have a reproduction repo here: https://github.com/justinfarrelldev/less-reproduction

There is one Less file in the Test module, and one in the styles folder as well. My goal is to have the @ import working as well of course.

Any help would be very much appreciated, been racking my brain against this for a few days haha.

Thank you!

**Comment by vankop** [comment, marked as answer]
you need to import it, like `import "./test.less"`

**Reply by justinfarrelldev** [reply]
> you need to import it, like `import "./test.less"`

While that is incredibly embarrassing, I added the import and it still didn't work. Pushed the changes to the repro

I do see "test.less" in the browser bundle now though and HMR does reload automatically for Less files, it just doesn't seem to actually apply styling

**Reply by vankop** [reply]
could you check `<style>` tag with your styles?

**Reply by justinfarrelldev** [reply]
> could you check `<style>` tag with your styles?

This answered it. I added this to my head: 
```
    <link rel="stylesheet" href="test.less" />
```

I guess there must have been some previous configuration that injected that tag from the setup I am porting this from, but as soon as I added it to this one, it just worked. Thank you!
